### ISSUE TITLE
🐛 Bug Report: Realtime Connections on Dashboard (Appwrite console)

### ISSUE THREAD

**Issue by s3ppo** [appwrite/appwrite#2462]
### 👟 Reproduction steps

after some time my appwrite is running my realtime connections does not show up on the dashboard (1 or 2 days)

what i can see in dockerlogs is this:
```
[Error] Type: Appwrite\Database\Exception\Authorization
[Error] Message: Missing action key: "write"
[Error] File: /usr/src/code/src/Appwrite/Database/Database.php
[Error] Line: 279
```

when i restart the realtime docker container its working again for 1 or 2 days

### 👍 Expected behavior

realtime connections should be shown on the dasboard

### 👎 Actual Behavior

realtime connections are always 0

### 🎲 Appwrite version

Version 0.11.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by TorstenDittmann** [comment]
Thank you for the report 👍🏻 

I'll be looking into this 🙏🏻

**Comment by s3ppo** [comment]
Thank you Torsten :)

i missed to copy a line from the container log.. the full log is:
```
[Error] Type: Appwrite\Database\Exception\Authorization
[Error] Message: Missing action key: "write"
[Error] File: /usr/src/code/src/Appwrite/Database/Database.php
[Error] Line: 279
```

**Comment by s3ppo** [comment]
solved since 0.12.x
THX ;-)
